pkg/tfbridge/assets_test.go

```diff
@@ -119,6 +119,5 @@ func TestFileArchives(t *testing.T) {
 	archive.Hash = ""
 	file4, err := t1.TranslateArchive(archive)
 	assert.Nil(t, err)
-	assert.Equal(t, file4.(string), "")
```

Comment:
Can we get an explanation? Very dangerous here.

Reply:
I think I dug into this and this is a degenerate case of an archive without a hash and without contents; the AWS bug I am tracking was about an archive with an actual hash but no contents, that doesn't seem to reproduce, but its' really tricky to simulate. Let's not hold off on this. Thanks!